src/apdu_sign.c

```diff
@@ -356,6 +356,7 @@ void parse_operation_inner(struct maybe_transaction *_U_ dest, bip32_path_t *_U_
         unsigned int inputs_len = MolReader_CellInputVec_length(&inputs);
 
         G.maybe_transaction.v.group_input_count = 0;
+        G.maybe_transaction.input_count = inputs_len; // for fallback code on parse failures.
```

Comment:
is it okay to use this input_count field? or should we have a fallback_input_count